exporters/dsexporter/dsexporter.go

```diff
@@ -1,20 +1,46 @@
 package main
 
 import (
-	"fmt"
 	"net/http"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
-func ping(w http.ResponseWriter, req *http.Request) {
-	fmt.Fprintf(w, "pong")
+// CustomExporter is a dummy exporter that implements the prometheus.Collector interface.
+// The Custom exporter is a way to structure our code so that it can generate and expose metrics.
+type CustomExporter struct {
```

Comment:
@avi-biton quick question: Since our focus should be on transitioning from a dummy service to a dummy exporter, I named it to be CustomExporter. Correct me if I am wrong.